src/routes/Root/Root.tsx

```diff
@@ -68,6 +72,52 @@ const Root = () => {
       <header role="banner" className="sticky top-0 z-10 print:hidden">
         <SkipLink hash="#jod-main" label={t('skiplinks.main')} />
         <NavigationBar
+          items={[
+            {
+              key: 'langSelector-key',
+              text: 'langSelector',
+              active: true,
+              component: ({ className }) => {
+                return (
+                  <div className="relative">
+                    <button
+                      type="button"
+                      className={`${className} bg-cover bg-center`}
+                      onClick={() => setLangMenuOpen(!langMenuOpen)}
+                    >
+                      <span className="material-symbols-outlined inline-flex select-none items-center leading-[140%] text-[24px] text-black bg-bg-gray rounded-full w-10 h-10 align-middle justify-center">
+                        language
+                      </span>
+                    </button>
+                    {langMenuOpen && (
+                      <div className="absolute right-0 min-w-max translate-y-6 transform">
+                        <PopupList>
+                          {supportedLanguageCodes.map((value) => {
+                            return (
+                              <PopupListItem key={`lang-${value}-key`}>
+                                <button
+                                  type="button"
+                                  className="text-bold text-black text-"
+                                  // eslint-disable-next-line @typescript-eslint/no-misused-promises
+                                  onClick={async () => {
```

Comment:
To get rid of this eslint-disable, create a function:

const changeLanguage = (value: string) => async () => {
    setLangMenuOpen(false);
    await i18n.changeLanguage(value);
    navigate(`/${i18n.language}`);
  };
And use here like so onClick={void changeLanguage(value)}

Reply:
Also, right now it will navigate to the front page. It would be nice to have current route in the url